test/tools/libtesteth/TestHelper.cpp

```diff
@@ -404,6 +404,39 @@ void checkCallCreates(eth::Transactions const& _resultCallCreates, eth::Transact
 	}
 }
 
+void runAllTestsInFolder(test::testType _type, const std::string& _folder)
+{
+	using path = boost::filesystem::path;
```

Comment:
In other places of the codebase, `using fs = boost::filesystem` is used.  Using the same name here would make the codebase more readable.

Reply:
like using fs = boost::filesystem 
ad fs::path()  instead path() ?

Reply:
yes.